ormconfig.env

```diff
@@ -4,6 +4,7 @@ TYPEORM_USERNAME = postgres
 TYPEORM_PASSWORD = password
```

Comment:
We shouldn't store credentials inside repo. I would create `ormconfig.env.example` file with some mock data to present just a sample of what we need to set.